tests/ui/test/features/search.feature

```diff
@@ -1,5 +1,13 @@
 Feature: Cloudfront tests
 
+@Test
```

Comment:
Can you remove this tag

Reply:
removed on [423c36c](https://github.com/NHSDigital/uec-cm/pull/80/commits/423c36c92e4e3b169d1bf4f7fa2b531700d9add6)